packages/tradershub/src/features/cfd/constants.tsx

```diff
@@ -72,37 +73,45 @@ export const MarketTypeDetails = (isEU?: boolean): TMarketTypeDetails => ({
         description:
             'Trade swap-free CFDs on MT5 with synthetics, forex, stocks, stock indices, cryptocurrencies and ETFs',
         icon: <SwapFreeMT5Icon />,
+        iconWithWidth: (width: number) => <SwapFreeMT5Icon width={width} />,
```

Comment:
@sergei-deriv why you are not passing iconWidth to this function instead of adding new key?
```suggestion
        export const MarketTypeDetails = (isEU?: boolean, iconWidth: number = 16): TMarketTypeDetails => ({
    all: {
        description:
            'Trade swap-free CFDs on MT5 with synthetics, forex, stocks, stock indices, cryptocurrencies and ETFs',
        icon: <SwapFreeMT5Icon width={iconWidth} />,
```

Reply:
@mahdiyeh-deriv I did it because I need the same `iconWithWidth` for `PlatformDetails` object, and there I can't pass any width to the object. So I've made decision to create methods with the same names in both. Let me know and I'll change `MarketTypeDetails`, but `PlatformDetails` I will leave the same...

Reply:
@mahdiyeh-deriv cause you are on vacation, let's leave it like this for now. I'll modify it in the next PRs if we really need it.